src/TextField.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import cx from 'classnames';
+
+/**
+ * @module TextField
+ */
+class TextField extends React.Component {
+
+	constructor(props) {
+		super(props);
+		this.state = { value: props.value };
+		this.handleChange = this.handleChange.bind(this);
+	}
+
+	handleChange(e) {
+		this.setState({ value: e.target.value });
+	}
+
+	render() {
+		const {
+			elId,
+			name,
+			label,
+			placeholder,
+			formAttrs,
+			children,
+			className,
+			...other
+		} = this.props;
+
+		const classNames = cx(
```

Comment:
Are you going to be adding custom classes? If not than you don't need to do this cx call